eas.json

```diff
@@ -33,6 +38,14 @@
     }
   },
   "submit": {
-    "production": {}
+    "production": {
+      "android": {
+        "track": "internal"
+      },
+      "ios": {
+        "ascAppId": "",
+        "appleTeamId": ""
+      }
+    }
```

Comment:
can you also add some links to docs for this section in the commit message please

Reply:
Updated with Commit document link